Changelog.md

```diff
@@ -18,6 +18,10 @@ Bugfixes:
  * Yul: Fix internal compiler error when a code generation error should be reported instead.
 
 
+Build system:
+ * Switch back to fully static binaries on Ubuntu.
```

Comment:
That's maybe a bit too inaccurate :-). I also just noticed that I actually never added a changelog entry for the switch to only semi-static builds either :-).
But it should rather say something like

```suggestion
 * Static release builds no longer depend on ``glibc``.
```
or something like that - it's not really about Ubuntu and it's not all Ubuntu binaries.
I'm not too happy about the phrasing of my own suggestion either, but something along those lines.

Reply:
Yeah that is fair enough :). Your phrasing is at least better than mine.

Reply:
I like that phrasing, nice. Fixed.